packages/camera/camera_android/lib/src/android_camera.dart

```diff
@@ -291,13 +300,17 @@ class AndroidCamera extends CameraPlatform {
   @override
   Stream<CameraImageData> onStreamedFrameAvailable(int cameraId,
       {CameraImageStreamOptions? options}) {
+    _installStreamController(onListen: _onFrameStreamListen);
+    return _frameStreamController!.stream;
+  }
+
+  void _installStreamController({Function()? onListen}) {
     _frameStreamController = StreamController<CameraImageData>(
-      onListen: _onFrameStreamListen,
+      onListen: onListen ?? () {},
```

Comment:
Consider https://github.com/flutter/plugins/pull/6290/files#r952963757 here as well.

Reply:
Replied in [this comment](https://github.com/flutter/plugins/pull/6290/#discussion_r957914180), I don't think it applies unless I'm misunderstanding something.